controllers/openstackcluster_controller_test.go

```diff
@@ -220,254 +215,10 @@ var _ = Describe("OpenStackCluster controller", func() {
 		Expect(err).To(BeNil())
 		scope := scope.NewWithLogger(clientScope, log)
 
-		computeClientRecorder := mockScopeFactory.ComputeClient.EXPECT()
-		computeClientRecorder.GetServer("bastion-uuid").Return(nil, gophercloud.ErrResourceNotFound{})
-
-		err = deleteBastion(scope, capiCluster, testCluster)
+		err = reconciler.deleteBastion(ctx, scope, capiCluster, testCluster)
 		Expect(err).To(BeNil())
 		Expect(testCluster.Status.Bastion).To(BeNil())
 	})
-	It("should adopt an existing bastion even if its uuid is not stored in status", func() {
```

Comment:
I want to investigate how to have unit testing coverage for these tests.

Reply:
but at the same time if we rework the bastion piece (see TODO) I'll wait a bit before writing them.